geomesa-accumulo/geomesa-accumulo-dist/src/main/assemblies/component.xml

```diff
@@ -20,6 +20,7 @@
                 <exclude>org.locationtech.geomesa:geomesa-geojson*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-blobstore*</exclude>
                 <exclude>org.locationtech.geomesa:geomesa-accumulo-compute*</exclude>
+                <exclude>org.locationtech.geomesa:geomesa-accumulo-spark-runtime*</exclude>
```

Comment:
Need to update the docs here http://www.geomesa.org/documentation/current/user/spark/accumulo_spark_runtime.html?highlight=spark